src/lab/Graphs/LineAreaGraph/styles.ts

```diff
@@ -7,7 +7,7 @@ interface StyleProps {
   align?: string;
 }
 
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles((theme: Theme) => ({
```

Comment:
kubera-chaos doesn't have theme support yet...don't use themes for graph components